scripts/integration.sh

```diff
@@ -12,6 +12,9 @@ sleep 2
 # Deploy title escrow creator to 0x4Bf7E4777a8D1b6EdD5F2d9b8582e2817F0B0953
 ./node_modules/@govtechsg/open-attestation-cli/dist/cjs/index.js deploy title-escrow-creator "test" -n local -k $account_key
 
+# Issue a verifiable document
+./node_modules/@govtechsg/open-attestation-cli/dist/cjs/index.js document-store issue --address 0x63A223E025256790E88778a01f480eBA77731D04 --hash 0xc9ea7d07be304e393e21f3b67f9e9f96b6bd829aab287af473b1d4f0924cbb81  -n local -k $account_key
+
```

Comment:
can rebase and issue from js directly liao

Reply:
ok updated

Reply:
sweet, throw away the .sh files too, no need liao